LEAF_Nexus/sources/Employee.php

```diff
@@ -1368,28 +1362,12 @@ public function search($input, $indicatorID = '')
 
                 break;
             //explicit search for disabled accounts
-            case substr(strtolower($input), 0, 18) === 'username_disabled:':
-                if ($this->debug)
-                {
-                    $this->log[] = 'Format Detected: Loginname';
-                }
-                $input = str_replace('username_disabled:', '', strtolower($input));
-                $searchResult = $this->lookupLogin($input, true);
-                break;
-            case substr(strtolower($input), 0, 9) === 'disabled:':
+            case substr(strtolower($input), 0, 18) === 'username.disabled:':
```

Comment:
Does this set of checks need to coordinate with the changes Jamie is putting into 4449?

Reply:
This raises an issue related to impact on [LEAF-4449](https://github.com/department-of-veterans-affairs/LEAF/pull/2493). The change affects how disabled accounts would be queried.

I would want to keep the scope of these two stories separate, since this one simply changes the user-facing syntax, while the other changes storage format on the backend.